### PR TITLE
Document differences between OSP13 and 16 for non-standard networking section

### PR DESCRIPTION
**Pull request by leifmadsen**
- Move to new header name
- Cleanup parameters
- Document differences between OSP13 and 16

### CODE REVIEW

**Comment by leifmadsen** [comment]
[Update Spine-Leaf network configuration section in STF documentation](https://trello.com/c/VRQe3DxJ/1091-update-spine-leaf-network-configuration-section-in-stf-documentation)

File: doc-Service-Telemetry-Framework/modules/proc_deploying-to-non-standard-network-topologies.adoc [hunk 1]
```diff
@@ -39,30 +39,81 @@ If you use {ProjectShort} with {OpenStack} {OpenStackVersion}  and plan to monit
...
+Additional leaf roles typically replace `_subnet` with `_leafN` where `N` represents a unique indentifier for the leaf.
++
+----
+ComputeLeaf1ExtraConfig:
+›   tripleo::profile::base::metrics::collectd::amqp_host: "%{hiera('internal_api_leaf1')}"
+›   tripleo::profile::base::metrics::qdr::listener_addr: "%{hiera('internal_api_leaf1')}"
+----
+endif::[]
 +
 On a CephStorage leaf role, an example configuration would be:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-to-non-standard-network-topologies.adoc:103]:
```suggestion
This example configuration is on a CephStorage leaf role:
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-to-non-standard-network-topologies.adoc [hunk 2]
```diff
@@ -39,30 +39,81 @@ If you use {ProjectShort} with {OpenStack} {OpenStackVersion}  and plan to monit
...
 
-. Add the following configuration example to the `ExtraConfig` section for each of the leaf roles. In this example, `internal_api0` is the value defined in the `name_lower` parameter of your network definition, and is the network to which the `Compute0` leaf role is connected. In this case, the network identification of 0 corresponds to the Compute role for leaf 0, and represents a value that is different from the default internal API network name.
+. Add the following configuration example to the `ExtraConfig` section for each of the leaf roles. In this example,
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-to-non-standard-network-topologies.adoc:50]: Well done on this section, looks like a bit of a nightmare to put together!

**Comment by leifmadsen** [comment]
> Just one change request!

Done!

**Pull request merged**